install_config/cluster_metrics.adoc

```diff
@@ -351,55 +351,64 @@ be added to your inventory file if it is necessary to override them.
+endif::[]
+ifdef::openshift-enterprise[]
+`openshift3/openshift-metrics-cassandra:3.5.0`, set prefix `openshift/openshift-`.
+endif::[]
 
 |`openshift_metrics_image_version`
-|The version for the component images. For
-example, with "openshift/origin-metrics-cassandra:v1.3", set version "v1.3".
+|The version for the component images. For example, with
+ifdef::openshift-origin[]
+`openshift/origin-metrics-cassandra:v1.3`, set version  as `v1.3`.
+endif::[]
+ifdef::openshift-enterprise[]
+`openshift3/openshift-metrics-cassandra:3.5.0`, set version as `3.5.0`.
+endif::[]
 
 |`openshift_metrics_startup_timeout`
-|The time in seconds to wait until Hawkular
-Metrics and Heapster starts up before attempting a restart.
+|The time, in seconds, to wait until Hawkular Metrics and Heapster start up
+before attempting a restart.
 
 |`openshift_metrics_duration`
 |The number of days to store metrics before they are purged.
 
 |`openshift_metrics_resolution`
-|The frequency that metrics are gathered defined
-as a number and time identifier seconds (s), minutes (m), hours (h).
+|The frequency that metrics are gathered. Defined as a number and time
+identifier: seconds (s), minutes (m), hours (h).
 
 |`openshift_metrics_cassandra_pvc_prefix`
-|The persistent volume claim prefix
-created for Cassandra. A serial number is appended to the prefix starting from
-1.
+|The persistent volume claim prefix created for Cassandra. A serial number is
+appended to the prefix starting from 1.
 
 |`openshift_metrics_cassandra_pvc_size`
 |The persistent volume claim size for each of the Cassandra nodes.
 
 |`openshift_metrics_cassandra_storage_type`
-|Use `emptydir` for ephemeral
-storage (for testing); `pv` to use persistent volumes, which need to be created
-before the installation; or `dynamic` for dynamic persistent volumes.
+|Use `emptydir` for ephemeral storage (for testing); `pv` for persistent volumes,
+which need to be created before the installation; or `dynamic` for dynamic
+persistent volumes.
 
 |`openshift_metrics_cassandra_replicas`
-|The number of Cassandra nodes for the metrics stack. There will be this many
-Cassandra replication controllers.
+|The number of Cassandra nodes for the metrics stack. This value dictates the
+number of Cassandra replication controllers.
 
 |`openshift_metrics_cassandra_limits_memory`
-|The amount of memory to limit the
-Cassandra pod, for example 2 GB. This value could be further adjusted by the
-start script based on available memory of the node on which it is scheduled.
+|The memory limit for the Cassandra pod. For example, a value of `2Gi` would limit
+Cassandra to 2 GB of memory. This value could be further adjusted by the start
+script based on available memory of the node on which it is scheduled.
 
 |`openshift_metrics_cassandra_limits_cpu`
-|The CPU limit for the Cassandra pod.
+|The CPU limit for the Cassandra pod. For example, a value of 4000m would limit
```

Comment:
4000m smooshed together is correct.  It is 4000 millicores if we need to spell it out.  $#@! k8s cpu limit units.